sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/ExposureInContactEpiDataSteps.java

```diff
@@ -0,0 +1,278 @@
+package org.sormas.e2etests.steps.web.application.contacts;
+
+import static org.sormas.e2etests.pages.application.contacts.EditEpidemiologicalDataContactPage.*;
+import static org.sormas.e2etests.pages.application.contacts.EditEpidemiologicalDataContactPage.EXPOSURE_DETAILS_KNOWN;
+import static org.sormas.e2etests.pages.application.contacts.EditEpidemiologicalDataContactPage.EXPOSURE_DETAILS_NEW_ENTRY;
+import static org.sormas.e2etests.pages.application.contacts.EditEpidemiologicalDataContactPage.LARGE_OUTBREAKS_AREA;
+import static org.sormas.e2etests.pages.application.contacts.ExposureNewEntryPage.*;
+
+import cucumber.api.java8.En;
+import javax.inject.Inject;
+import javax.inject.Named;
+import org.assertj.core.api.SoftAssertions;
+import org.sormas.e2etests.helpers.WebDriverHelpers;
+import org.sormas.e2etests.pojo.web.ExposureDetails;
+import org.sormas.e2etests.pojo.web.ExposureInvestigation;
+import org.sormas.e2etests.services.ExposureDetailsService;
+import org.sormas.e2etests.services.ExposureInvestigationService;
+import org.sormas.e2etests.state.ApiState;
+
+public class ExposureInContactEpiDataSteps implements En {
+
+  private final WebDriverHelpers webDriverHelpers;
+  private static ExposureInvestigation exposureInvestigationInput;
+  private static ExposureDetails exposureDetailsInput;
+  private String EPIDATA_FOR_LAST_CREATED_CONTACT_URL;
+
+  @Inject
+  public ExposureInContactEpiDataSteps(
+      WebDriverHelpers webDriverHelpers,
+      ExposureInvestigationService exposureInvestigationService,
+      ExposureDetailsService exposureDetailsService,
+      ApiState apiState,
+      @Named("ENVIRONMENT_URL") String environmentUrl) {
+    this.webDriverHelpers = webDriverHelpers;
+
+    When(
+        "I am accessing the Epidemiological tab using of created contact via api",
+        () -> {
+          webDriverHelpers.waitForPageLoaded();
```

Comment:
this shouldn't be here. If the method will be used in other tests in another logic, there's no need to wait for the page to be loaded. I suggest moving it in the previous called step/method

Reply:
addressed.